llvm/utils/TableGen/ARMTargetDefEmitter.cpp

```diff
@@ -0,0 +1,67 @@
+//===- ARMTargetDefEmitter.cpp - Generate data about ARM Architectures ----===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This tablegen backend exports information about CPUs, FPUs, architectures,
+// and features into a common format that can be used by both TargetParser and
+// the ARM backend.
```

Comment:
"the ARM and AArch64 backends."

Normally it's fine to say ARM for something that works for both, because the world outside Arm refers to both that way anyway. If the comment is talking about llvm's code structure though, better to list both names.